FileProcessor/src/main/java/com/group7/FileProcessor/ScenarioMaker.java

```diff
@@ -8,61 +8,45 @@
 import com.group7.FileProcessor.pojo.*;
 import java.io.IOException;
 
-
 /**
  *
  * @author keric
  */
-public class Scenario {
+public class ScenarioMaker {
+
+    private String testScenario;
+    public ScenarioPOJO scenario = new ScenarioPOJO();
+
+    public ScenarioMaker(String testScenario) {
+        this.testScenario = testScenario;
 
-    public static void main(String[] args) throws IOException {
-           
+        try {
+            JsonNode node = Json.parse(testScenario);//make JsonNode
+            scenario = Json.fromJson(node, ScenarioPOJO.class); //map to class
+        } catch (IOException e) {
+            System.out.println("ERROR: IOException when reading parsing and reading the scenarioAsString in Sequencer.java\n" + e);
+        }
+    }
 
-String testScenario = "{\n" +
-"    \"name\": \"Test Scenario\",\n" +
-"    \"processing_elements\": [     \n" +
-"        {\n" +
-"            \"type\": \"Print\", \n" +
-"            \"input_entries\": [\n" +
-"                {\n" +
-"                    \"type\": \"remte\",\n" +
-"                    \"repositoryId\": \"r-34w6\",\n" +
-"                    \"entryId\" : \"1\"\n" +
-"                }\n" +
-"            ],\n" +
-"            \"parameters\": [\n" +
-"                {\n" +
-"                    \"name\": \"Max\",\n" +
-"                    \"value\": \"100\"\n" +
-"                }\n" +
-"            ]           \n" +
-"        }\n" +
-"    ]\n" +
-"}";
-        
-        JsonNode node = Json.parse(testScenario);//make JsonNode
-        ScenarioPOJO scenario = Json.fromJson(node,ScenarioPOJO.class);//map to class
-        
+    public void print() {
```

Comment:
is there a reason this class gets a print() method added that looks the same as the print method in ScenarioPOJO.java?

Reply:
Nope, that class is fully obsolete.  I was waiting a bit to delete it, just to be sure.